trin-core/src/types/accumulator.rs

```diff
@@ -23,23 +23,23 @@ use crate::utils::bytes::{hex_decode, hex_encode};
 pub const EPOCH_SIZE: usize = 8192;
 
 /// Max number of epochs - 2 ** 17
-const MAX_HISTORICAL_EPOCHS: usize = 131072;
+const _MAX_HISTORICAL_EPOCHS: usize = 131072;
```

Comment:
What is the reason for prefixing this with `_` instead of deleting the constant?

Reply:
We reference the constant in the line below, so I thought to leave it in to have the actual int as a reference. But yeah, it is unnecessarily compiled, so I've commented it out